SPEC.md

```diff
@@ -310,6 +311,30 @@ Optional environment parameters:
 
 All parameters, with the exception of `CNI_PATH`, must be the same as the corresponding `ADD` for this container.
 
+
+#### `STATUS`: Check plugin status
+`STATUS` is a way for a runtime to determine the readiness of a network plugin.
+
+A plugin must exit with a zero (success) return code if the plugin is ready to service ADD requests. If the plugin is not able to service ADD requests, it must exit with a non-zero return code and output an error on standard out (see below).
+
+The following error codes are defined in the context of `STATUS`:
+
+- 50: The plugin is not available (i.e. cannot service `ADD` requests)
+- 51: The plugin is not available, and existing containers in the network may have limited connectivity.
```

Comment:
After discussing this forever in the maintainers' meeting, we're not sure if we should actually include this.

Reply:
I was suggesting to start small without specific error codes and iterate on feedback and use cases, agree with Dan that is hard to know in advance which code is better suit for what scenario or more important, what to do in each case on the upstream implementations